core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,28 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+.popover{
+  font-size: 13px;
+  max-width: 300px;
+  text-align: center;
+  z-index: -1;
+  color: white;
+  background-color: black;
+}
+.popover.bottom > .arrow{
+  top: -11px;
+  left: 50%;
+  margin-left: -11px;
```

Comment:
@shaz13 I just want to tell about why i have included these two classes again here so the reasons are:
1. .popover class is over writing the thirdparty.css files's popover class through which i am changing the text color and background color of popover you are working on.
2. But the changes done above in no. 1 reason are not enough for changing color of tooltip arrow pointing the top of bottom type of help popover. thus i overwrite that classes again here so that the popover become uniform in color combination as suggested.
Thus, only color based property changes are not satisfying the need of this issue those two classes have to be included here.